packages/@okta/vuepress-site/docs/concepts/role-assignment/index.md

```diff
@@ -0,0 +1,107 @@
+| `ORG_ADMIN`                             | Organizational Administrator        |                                       |
+| `READ_ONLY_ADMIN`                       | Read-Only Administrator             |                                       |
+| `REPORT_ADMIN`                          | Report Administrator                |                                       |
+| `SUPER_ADMIN`                           | Super Administrator                 |                                       |
+| `USER_ADMIN`                            | Group Administrator                 | [Groups](/docs/reference/api/groups/) |
+
+Standard role assignment is done in two steps:
+1. Assign a role to a user or group. At this point the admin has the supported privileges fo the role over all resources across organization.
+2. (Optional) If the role supports targets, use one of the [target operations](/docs/reference/api/roles/#role-target-operations) to specify which specific resource the admin can manage.
+
+Note that the entities involved in standard role assignment are:
+* a role; identified either by type or id returned from [listing API](/docs/reference/api/roles/#list-roles)
+* a principal; either a group or a user
+* (Optional) a resource; when using target [target operations](/docs/reference/api/roles/#role-target-operations) this could be either an app or a group
+
+## Custom Role Assignment
+Custom roles can be built by piecing [permissions](/docs/reference/api/roles/#permission-types) together. Once a custom role is built, you can use its id or name to assign to admins. The process is:
+1. Build a custom role
+2. Build a set of resources
+3. Bind the admin with the role from step 1 targeting the resource set from step 2
+
+An assignment of a role to an admin is called a [binding](/docs/reference/api/roles/#binding-object). Conceptually, a binding is identified by the role id. A maximum of one role per resource set will be allowed. Assignment of more admins to the same role targetting the same resource set is possible and they are added as [members]((/docs/reference/api/roles/#member-object)) of the same binding. Hence a binding could simply be seen as:
+```json
+{
+  "${roleId}": [
+    "${memberId1}",
+    "${memberId2}",
+    ...
+  ]
+}
+```
+
+Therefore, when dealing with custom roles, three entities are always in play:
+* a role; identified by its name or id
+* a principal; either a group or a user
+* a resource-set; identified by its id
+
+### Resource Sets
+Custom roles can only be assigned if scoped to a set of recourses from the beginning. In other words, you need to have a resource set in order to be able to assign a custom role.
+
+A resource set is simply a collection of resources. The following resources are currently supported:
+* All users
+* All groups
+* A specific group
+* All users within a specific group
+Note that the distinction between users and groups allows for creating custom admins that are only scoped to users or groups.
```

Comment:
This distinction is because 'user' permissions are for performing actions on all user resources, or users within a specific group, and the 'group' permissions can be used to perform actions on all groups resources or specific groups

Reply:
@bethwang-okta is right - however, I don't know that we want to bring in the applicability of a given permission here - i.e. keep this section focused on what a resource set is, and what resources can fall within it.  individual permissions and their applicability based on what resources are targeted belongs elsewhere perhaps?

Reply:
I now see this in a tip/example down below in Resource Identifiers section, so am ok removing from here, however  this note still seems unclear/doesn't serve a purpose as written.

Reply:
dropped this Note from here to avoid confusion